ibm_catalog.json

```diff
@@ -0,0 +1,103 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-icd-redis",
+      "label": "ICD Redis Module",
+      "product_kind": "module",
+      "tags": [
+        "dev_ops",
+        "target_terraform",
+        "terraform",
+        "module"
+      ],
+      "keywords": [
+        "terraform",
+        "redis",
+        "icd"
+      ],
```

Comment:
add "database" for searching?

Reply:
perhaps we also add other redis capabilities such as `cache` and `message-broker` ?

Reply:
@rajatagarwal-ibm You may have missed my comment?

Reply:
my bad, perhaps didn't refresh my browser. Addressed